src/hooks/useSearch.js

```diff
@@ -0,0 +1,22 @@
+import { useState } from 'react'
+import axios from 'axios'
+
+export const useSearch = () => {
+  const [hotels, setHotels] = useState([])
+
+  const search = async ({ city, startDate, endDate, single, double }) => {
+    const response = await axios.get(global.API_BASE_URL + 'api/hotels', {
+      params: {
+        city,
```

Comment:
Is params the same as query? 
Moreover properties below shouldnt be uncommented?